server/src/main/java/com/linecorp/centraldogma/server/command/AbstractPushCommand.java

```diff
@@ -81,7 +81,7 @@ public String detail() {
     }
 
     /**
-     * Returns the {@link Markup} of the {@link #detail()}.
+     * Returns the {@link Markup} of the {@code detail()}.
```

Comment:
Maybe revert? This class still have `#detail()`

Reply:
Revert and make `AbstractPushCommand` `public` so that Javadoc is correctly rendered.